packages/desktop-client/src/components/budget/SidebarCategory.tsx

```
@@ -54,12 +65,25 @@ export function SidebarCategory({
+  const navigate = useNavigate();
+
+  const { schedule, status: scheduleStatus } = useCategoryScheduleGoalTemplate({
+    category,
+  });
+  const isScheduleUpcomingOrMissed =
+    scheduleStatus === 'missed' ||
+    scheduleStatus === 'due' ||
+    scheduleStatus === 'upcoming';
+
+  const isScheduleRecurring =
+    schedule && schedule._date && !!schedule._date.frequency;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)